scripts/deploy-beta-apm.js

```diff
@@ -52,6 +52,7 @@ module.exports = async callback => {
   console.log('=========')
   const apmAddr = receipt.logs.filter(l => l.event == 'DeployAPM')[0].args.apm
   console.log('deployed APM:', apmAddr)
+  console.log(apmAddr)
```

Comment:
What is the point of logging this again (and in [the ENS script](https://github.com/aragon/aragonOS/pull/243/files#diff-1fb706e498588d59d326e44987cc814dR15))?

Should we return the value instead so users of the script can have access to the deployed address?

Reply:
So we can easily pipe the value in a bash script https://github.com/aragon/aragon-apps/pull/115/files#diff-273eaaae9a8f6fc623375ffa23d33531R6